epitator/count_annotator_2.py

```diff
@@ -0,0 +1,209 @@
+from epitator.annodoc import AnnoDoc
```

Comment:
The epitator part is unnecessary on all these imports. It should be `from .annodoc import AnnoDoc`

Reply:
Changed.